.gitignore

```diff
@@ -34,3 +34,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+package-lock.json
```

Comment:
Don't think we need these in gitignore, we use yarn.lock here, also we already have this `.env*.local`